httpTriggerAutoScale/httpTriggerAutoScale/httpTriggerAutoScale.csproj

```diff
@@ -7,11 +7,9 @@
     <Compile Remove="Utils.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="0.10.1-preview" />
-    <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="3.0.3" />
+    <PackageReference Include="Azure.Data.Tables" Version="12.0.0-beta.6" />
```

Comment:
Please update to use the latest stable release. At the time of writing, that's version 12.3.0.

Reply:
@scottaddie, I have updated, please review.